packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -1776,6 +1809,7 @@ export function attach(
       pushOperation(ownerID);
       pushOperation(displayNameStringID);
       pushOperation(keyStringID);
+      pushOperation(isErrorBoundary(fiber) ? 1 : 0);
```

Comment:
Do we need to add this bit for every Fiber that mounts? We _can_ if we need to, but it's maybe heavier than we need. (I think we could wait until we're _inspecting_ a component to send this information.)

_If we do need to add this here_, then we also need to update a few other places to keep them in sync:
* `onBridgeOperations` in `react-devtools-shared/src/devtools/store.js` (already done)
* `onBridgeOperations` in `react-devtools-shared/src/devtools/ProfilerStore.js`
* `updateTree` in `react-devtools-shared/src/devtools/views/Profiler/CommitTreeBuilder`
* `printOperationsArray` in `react-devtools-shared/src/utils`
* The `OVERVIEW.md` doc that describes what each message includes

But I _think_ we can avoid sending this bit eagerly for every component, and instead only send it over when we're _inspecting_ a Fiber. Then we can add a field to the inspected payload that is like `nearestErrorBoundaryID`. Does this make sense?